common/countryRole.ts

```diff
@@ -32,7 +32,7 @@ export const roles = {
 
 export const roleKeys = R.pipe(R.values, R.map(R.prop('role')))(roles)
 
-export const getRoleLabelKey = (roleName: any) => R.path([roleName, 'labelKey'], roles)
+export const getRoleLabelKey = (roleName: any): string => R.path([roleName, 'labelKey'], roles)
```

Comment:
Should we remove ramda and use ex. roles?.[roleName]?.labelKey

Reply:
future refactor yes